src/components/Button/styles.js

```diff
@@ -11,24 +11,21 @@ export const StyledButton = styled.button`
   padding: 0.6em 1em;
   font-family: ${({ theme }) => theme.fonts.families.sourceSans};
   font-size: ${({ theme }) => theme.fonts.sizes.medium};
-  border: none;
+  border: solid 1px
+    ${({ light, theme }) => (light ? theme.colors.light : theme.colors.dark)};
   border-radius: ${({ theme }) => theme.globals.radius};
-  cursor: pointer;
+  cursor: ${({ isLoading }) => (isLoading ? 'wait' : 'pointer')};
   opacity: ${({ isLoading }) => (isLoading ? 0.5 : 1)};
 
   > ${StyledIcon} {
     margin-right: ${({ theme }) => theme.sizes.tiny};
   }
 
-  ${({ isLoading }) =>
-    !isLoading
-      ? css`
-          &:hover {
-            background: ${({ light, theme }) =>
-              light ? theme.colors.accentLight : theme.colors.accentDark};
-          }
```

Comment:
Where are those styles, now ☝️ ?

Reply:
Nowhere, I removed them since the ternary condition is outdated (because I added other button styles). 
Since we don't have any UI styles for the hover state at the moment, I will add  these styles later on.

Reply:
OK